eventespresso/event-espresso-core
# Add Admin Options and Conditional Loading for EDTR

## Problem this Pull Request solves
Currently the display of the new EDTR is hardcoded into the Event Admin and the existing "Event Tickets and Datetimes" metabox is still displayed. 
This branch adds a new section to the **Events Admin Default Settings** tab that allows the EDTR to be turned on and off, as well as set a couple of options for setting default values for how the Entity Lists function. Filters have also been added so that the REM addon will be able to force usage of the EDTR.

## testing

* log into the admin and visit the **Event Espresso - Events** > **Default Settings** admin page tab
* [ ] confirm that new setting options appear at the top of the form
* change the first option to "Advanced Editor" and click save
* click on the **Overview** tab so that it opens in another browser tab (or window) and click on an event to open its editor
* [ ] confirm that the new EDTR loads and that the old "Event Tickets and Datetimes" metabox does NOT appear
* back in the first tab, set the "Activate Advanced Editor?" input to "Legacy Editor" and click save
* refresh the event editor in the second tab
* [ ] confirm that the new EDTR does NOT load and that the old "Event Tickets and Datetimes" metabox appears again
* try changing some settings for the dates or tickets using the legacy editor and click save
* [ ] confirm that the saving the options was successful and that no errors occurred
* back in the first tab, play around with the "View" and "Per Page" options for the EDTR and click save
* refresh the event editor in the second tab
* [ ] confirm that your selections have been reflected in the appearance of the EDTR
